### DISCUSSION TITLE
Xbox Game Pass Support to install PC titles

### DISCUSSION THREAD

**Discussion by julianrios** [moraroy/NonSteamLaunchers-On-Steam-Deck#388] [Ideas]
Not sure if this is possible but if Xbox has an installer that can run an exe would be cool to see support for this

**Comment by moraroy** [comment]
@julianrios would need to first make sure the xbox app runs on linux haha

**Reply by julianrios** [reply]
haha yea meant to add this as a feature request but seems microsoft has this locked down on their app store

**Comment by lyndon025** [comment]
I would like to +1 to this suggestion, now that we have a Windows version of NSL (add Xbox App games to NSL on windows)

**Reply by moraroy** [reply]
Is that not working? Run a scan

**Reply by lyndon025** [reply]
> Is that not working? Run a scan

i apologize. I saw Xbox App not on the list of supported launchers so i assumed its not supported yet
